base_agent/loco_mc_agent.py

```diff
@@ -156,7 +156,7 @@ def handle_exception(self, e):
         # if so, we raise a reasonable message to the user, and then do some clean
         # up and continue
         if isinstance(e, ErrorWithResponse):
-            self.send_chat("Oops! I got confused and wasn't able to complete my last task :(")
+            self.send_chat("Oops! Ran into an exception.\n'{}''".format(e.chat))
```

Comment:
overall, I like the idea of sending across detailed error messages to the dashboard